### ISSUE TITLE
 PROPOSAL: ROCKMONGO TO NEW GITHUB ORGANIZATION

### ISSUE THREAD

**Issue by stefanocudini** [iwind/rockmongo#133]
**?!The question we are all about is.. the project is alive or dead!?** 

However it is very unpleasant this situation I'm a happy fan of this good software!

We ask to the author @iwind  if it is willing to make a fork or move it into a new github organization and continue the development with the help of everyone!

**Comment by woodworker** [comment]
why not just fork?
when was the last time the author answered a bugticket in this project?

fork it and put the new organization link in here.

**Comment by saurabheights** [comment]
It's good etiquettes to get the permission of the original author. He **might be** willing to add contributors. We should wait for him to respond and if there is no response(say for 2-4 weeks), then only create a fork.

**Comment by stefanocudini** [comment]
exactly

**Comment by stefanocudini** [comment]
https://github.com/iwind/rockmongo/blob/master/LICENSE.txt#L10

It would appear that the license does not allow you to create an organization with the same name
If any of you are licensed, please give me suggestions

**Comment by jakoch** [comment]
One more thing to consider: MongoDB, Inc. is talking to companies/projects whose tools use the words Mongo or MongoDB in their names, about changing what they are called. That's the reason for the recent renames of "MongoChef to Studio 3T" and ["RoboMongo to Robo 3T"](http://blog.robomongo.org/changing-the-name-of-robomongo/). 
So, when you are finally clear to fork, don't forget to rename it.

**Comment by woodworker** [comment]
When forking, maybe also renaming it also?

Here is a list of Humongous Rocks ;0
http://www.touropia.com/largest-monoliths-in-the-world/

**Comment by Masadow** [comment]
So, what's the situation, has anyone forked so far ? :)

I would be glad to help bugfixing, especially the #29 that could be so harmful !

**Comment by woodworker** [comment]
i hereby declare this project dead

**Comment by beesofts** [comment]
We use Robo3T for month and are very happy with this

**Comment by stefanocudini** [comment]
hi @beesofts this is a very different thing rockmongo for some use cases is better with its web interface

**Comment by beesofts** [comment]
@stefanocudini I agree with you but I havent found any living project similar to rockmongo

**Comment by woodworker** [comment]
as i already said last year, Fork it, give it a new name and then announce the new Name/URL in this issue
or nothig will ever happen

**Comment by stefanocudini** [comment]
@woodworker I was thinking of doing it! do you think could there be a sufficient number of contributors for a new project?
